deepface/modules/representation.py

```diff
@@ -122,11 +122,12 @@ def represent(
         confidence = img_obj["confidence"]
 
         # resize to expected shape of ml model
-        img = preprocessing.resize_image(
-            img=img,
-            # thanks to DeepId (!)
-            target_size=(target_size[1], target_size[0]),
-        )
+        if target_size is not None:
```

Comment:
yolo's target size is set to none, and that is why you are going to miss those functionalities.

we should set a target size e.g. (224, 224) for it.

- https://github.com/serengil/deepface/blob/master/deepface/modules/preprocessing.py#L116
- https://github.com/serengil/deepface/blob/master/deepface/modules/preprocessing.py#L118